AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -3468,6 +3440,27 @@ - (ASLayout *)measureWithSizeRange:(ASSizeRange)constrainedSize
   return [self layoutThatFits:constrainedSize parentSize:constrainedSize.max];
 }
 
+- (void)setPreferredFrameSize:(CGSize)preferredFrameSize
+{
+  ASDN::MutexLocker l(__instanceLock__);
+
+  // Deprecated preferredFrameSize just calls through to set width and height
+  self.width = ASDimensionMake(preferredFrameSize.width);
+  self.height = ASDimensionMake(preferredFrameSize.height);
+  [self invalidateCalculatedLayout];
```

Comment:
With the changes we will do in #2261 this has to change in any case. For now I think it's fine to do it this way to have the same behavior as before.